packages/metal-state/src/State.js

```diff
@@ -304,6 +304,8 @@ class State extends EventEmitter {
 	 */
 	emitBatchEvent_() {
 		if (!this.isDisposed()) {
+			this.context_.emit('stateWillChange', this.scheduledBatchData_);
```

Comment:
Not related to this change in particular, but I'm wondering how can we document what events are emitted so one knows what to listen :)